cbsurge/core/variable.py

```diff
@@ -17,12 +17,14 @@
 from pydantic import BaseModel, FilePath
 from pyogrio import read_info
 from rich.progress import Progress
+from sqlalchemy.testing.plugin.plugin_base import options
```

Comment:
@Thuhaa sqlalchemy code looks not used anywhere. you maybe can delete it. because of this code, CI looks failing